messaging/methods/messages/getMessages.md

```diff
@@ -147,7 +135,8 @@ Link: <https://messaging.bandwidth.com/api/v2/users/{accountId}/messages?message
             "messageType":"sms",
             "segmentCount":1,
             "errorCode":9902,
-            "receiveTime":"2020-04-07T14:03:07.000Z"
+            "receiveTime":"2020-04-07T14:03:07.000Z",
+            "carrierName":"unknown"
```

Comment:
can you make carrierName value as `other` instead of `unknown`

Reply:
Done